sentry-ruby/lib/sentry/transaction.rb

```diff
@@ -0,0 +1,34 @@
+module Sentry
+  class Transaction < Span
```

Comment:
In this function we need a static function called `from_sentry_trace` that gets a header string and creates a Transaction with `traceid` and so on, see: https://github.com/getsentry/sentry-php/pull/1134/files#diff-b778af5cbead054606b23aed4835bd172616ebd9ab923275fe5845e6b45b4b9bR77